portfolio/src/main/webapp/index.html

```diff
@@ -35,6 +35,11 @@ <h4> Fun fact about me: </h4>
       <p> I love In-N-Out! </p>
       <img src="/images/innout.jpg" width="300" height="300"/>
 
+      <!-- Servlet response -->
+      <p><br/></p>
```

Comment:
Is the `<p>` tag necessary? Does it work if it's just `<br/>`?

Reply:
I'll look into this, thanks